generate-wasm-examples/build_wasm_examples_debug.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+# locally build wasm examples for testing
+# usage:
+#   ./build_wasm-examples_debug.sh 6 # builds the first 6 examples
+
+if ! command -v wasm-bindgen >/dev/null 2>&1
+then
+    echo "wasm-bindgen could not be found - install it using: cargo install wasm-bindgen-cli"
+    exit 1
+fi
+
+if [ -z "$1" ]
+  then
+    echo "Missing example amount, use: './build_wasm_examples_debug.sh 5' to build the first 5 examples"
+    exit 1
+fi
+
+# Switch to script's directory, letting it be called from any folder.
+cd $(dirname $0)
+
+# generate_wasm_examples will clone the bevy repo to ./bevy by default.
+# If you want to use your fork or local changes, clone, symlink or copy your own bevy repo/folder:
+# git clone https://github.com/<username>/bevy bevy
+# cp ~/projects/bevy ./bevy
+./generate_wasm_examples.sh --no-pull
+
+cd bevy
+
+git reset HEAD --hard
+CARGO_PROFILE_RELEASE_OPT_LEVEL='z' CARGO_PROFILE_RELEASE_DEBUG="true" cargo run -p example-showcase -- --per-page $1 --page 0 build-wasm-examples --content-folder ../../static/assets/examples/wasm_webgl2 --api webgl2 --website-hacks
+CARGO_PROFILE_RELEASE_OPT_LEVEL='z' CARGO_PROFILE_RELEASE_DEBUG="true" cargo run -p example-showcase -- --per-page $1 --page 0 build-wasm-examples --content-folder ../../static/assets/examples/wasm_webgpu --api webgpu
```

Comment:
IMO, it's a bit unfortunate to repeat these invocations here in this new script. Would prefer that `generate_wasm_example.sh` handle the limiting.

Also, I think I or someone may have added a `--debug` option to `example-showcase`.

Reply:
but generate_wasm_examples.sh never calls `example-showcase build-wasm-example`? only the CI yaml has another version of this.

Reply:
> Also, I think I or someone may have added a --debug option to example-showcase.

Just tried this: the option does exist, but its not implemented for the `build-wasm-example` command, only for the `run` command. See: https://github.com/bevyengine/bevy/blob/0b8844a3a060b2a5b7c3302bd951e39fbf6aa1c3/tools/example-showcase/src/main.rs#L723

Reply:
ah right, sorry!